community/cloud-foundation/templates/firewall/firewall.py.schema

```diff
@@ -24,21 +24,21 @@ properties:
   network:
     type: string
     description: |
-      The network name. Defaults to 'global/networks/default'
+      The network name. Defaults to 'global/networks/default'.
   rules:
     type: array
     description: |
       An array of firewall rules as defined in the documentation:
-      https://cloud.google.com/compute/docs/reference/rest/beta/firewalls
+      https://cloud.google.com/compute/docs/reference/rest/beta/firewalls.
 
-      If the 'priority' field is specified in the rule, it's always used as is.
-      If the 'priority' field for the rule is not set, the template will set
-      the priority to the same value as index of the rule in the array +1000.
-      For example, if not set in rule, priority for the first rule in the array
-      is '1000', for the second rule '1001', and so on. What this means is that
-      if the 'priority' field isn't specified in any of the rules in the array,
-      the ruleset will be automatically sorted by priority, so consistency when
-      specifying (or not specifying) the 'priority' field is strongly advised.
+      If the 'priority' field value is set in a rule, that value is used "as is".
+      If the 'priority' field value is not set in the rule, the template sets
+      the priority to the same value as the rule's index in the array +1000.
+      For example, the priority for the first rule in the array becomes '1000', 
+      for the second rule '1001', and so on. If the 'priority' field is not set in 
+      any of the rules in the array, the ruleset is sorted by priority automatically. 
+      We strongly advise to consistenly setting (or not setting) the 'priority' field 
```

Comment:
> We strongly advise to consistenly setting (or not setting) the 'priority' field

We should both try to make this clearer and make it stand out a bit more...
what about something like... : 

"We strongly advise being consistent when using the 'priority' field. It's preferred to either use it throughout the ruleset or avoid it entirely."

...maybe with a better wording :)

Reply:
We strongly advise being consistent in your use of the 'priority' field: either provide or skip values in all instances throughout the ruleset.